src/components/dialog/dialog.d.ts

```diff
@@ -2,6 +2,8 @@ import * as React from "react";
 import Modal, { ModalProps } from "../modal/modal";
 
 export interface DialogProps extends ModalProps {
+  /** Dialog content */
+  children?: React.ReactNode;
```

Comment:
If we're extending `ModalProps` and that has `children` on it, we don't need this

Reply:
Good spot. I worked backwards from `Alert` through `Dialog` to `Modal` so it didn't occur to me 😶‍🌫️